app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -92,17 +93,35 @@ class HomeMenu(
         }
     }
 
+    val syncedTabsItem = BrowserMenuImageText(
+        context.getString(R.string.synced_tabs),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor
+    ) {
+        onItemTapped.invoke(Item.SyncedTabs)
+    }
+
+    private var signedInToFxa = false
     private fun getSyncItemTitle(): String {
         val authenticatedAccount = accountManager.authenticatedAccount() != null
         val email = accountManager.accountProfile()?.email
 
         return if (authenticatedAccount && email != null) {
+            signedInToFxa = true
```

Comment:
Function with side effects is not a good idea,  for `signedInToFxa` to be correct `getSyncItemTitle()` should be executed in an specific order before `signedInToFxa` gets used, that could be fragile as `getSyncItemTitle()` main purpose is not getting the Sync status. I think we could have a function that queries the sync status and can be called from both places L110 and L122.